.github/workflows/test-suite.yaml

```diff
@@ -48,7 +48,6 @@ jobs:
     - name: Build RKE2 Binary and Runtime Image
       run: |
         GOCOVER=true make package-bundle
-        make package-image-runtime
```

Comment:
I think we need another target that just builds a single rke2-runtime tarball, without all the other garbage.